packages/database/tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "sourceMap": true
   },
   "extends": "@cypherock/tsconfig/node.json",
-  "include": ["./src/**/*"],
+  "include": ["./src/**/*", "tests/encryptDbTests.ts"],
```

Comment:
Is this needed?

Reply:
No, the file is moved to another folder so this is not required